tox.ini

```diff
@@ -400,7 +400,7 @@ commands =
 basepython = python3.8
 deps = -r{toxinidir}/requirements/docs
 commands =
-    sphinx-build -W -E -b html docs docs/_build/html []
+    sphinx-build -j auto -W -E -b html docs docs/_build/html []
```

Comment:
Shaved 10s off my local build time, so thought I'd just leave it in.

Reply:
I usually use the makefile